### ISSUE TITLE
CSS Theming

### ISSUE THREAD

**Issue by Hedronmx** [vjba/whatstron#9]
Will it be possible to add CSS Theming? I am willing to create several css themes.

**Comment by vjba** [comment]
CSS injection is possible. I'll open a PR for the feature for you to contribute any themes.

**Comment by vjba** [comment]
branch with some skeleton code available [here](https://github.com/vjba/whatstron/tree/css-injection)

**Comment by Hedronmx** [comment]
Alright Ill fork and clone the branch when I can.

Any themes you would want? I want to create css for the specific os, so it feels more "native"

**Comment by vjba** [comment]
Great idea @Hedronmx!

Start with what is easiest first as to not waste your time. Re-styling the css on whatsapp web seems a big time investment with so much going on. Maybe begin with a trial style sheet for testing purposes? 

Once it's working it would be great if you could develop some sheets for the major themes like Elementary OS, Ubuntu, and Flat Remix.

**Comment by Hedronmx** [comment]
Yeah Ill start little by little, hopefully whatsapp wont change the style in an update lol!
